src/item.h

```diff
@@ -896,8 +896,6 @@ class item : public JsonSerializer, public JsonDeserializer
  typedef std::vector<item> t_item_vector;
  t_item_vector components;
 
- item clone(bool rand = true);
```

Comment:
Yes, but that is only part of the problem. Even without this parameter, no further properties of the current item are copied into the new item. For example `rot`, `last_rot_check`, `fridge` are completely ignored. The new item will be fresh.